features/u.yml

```
@@ -1,7 +1,6 @@
-draft_date: 2024-06-05
 name: <u>
-description: The `<u>` element TODO.
+description: The `<u>` HTML element makes text appear with an underline.
```

Perhaps even simpler, but also accounting for things other than text:

```suggestion
description: The `<u>` HTML element underlines text and other content.
```